.gitignore

```diff
@@ -0,0 +1 @@
+*.sublime-*
```

Comment:
This should be in your global gitignore (see https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) instead as this isn’t project-specific.